### PR TITLE
Added build yml for github actions

### PR DESCRIPTION
**Pull request by rmathew1011**

### CODE REVIEW

**Comment by kaladay** [comment]
Have you checked to see if this test failure is resolvable?
```
Chrome Headless 118.0.5993.117 (Linux x86_64) ERROR
  An error was thrown in afterAll
  ReferenceError: module is not defined
      at tests/unit/controllers/userRepoControllerTest.js:5:2
      at <Jasmine>
      at tests/unit/controllers/userRepoControllerTest.js:1:1
```

At glance, this seems like something that can be fixed.

File: app/components/version/version-directive_test.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+  //   it('should print current version', function() {
+  //     module(function($provide) {
+  //       $provide.value('version', 'TEST_VER');
+  //     });
+  //     inject(function($compile, $rootScope) {
+  //       var element = $compile('<span app-version></span>')($rootScope);
+  //       expect(element.text()).toEqual('TEST_VER');
+  //     });
+  //   });
+  // });
```
Comment (kaladay) [inline app/components/version/version-directive_test.js:16]: We should avoid the practice of commenting out code.
Just delete.
But in this case, I am more interested in why it is failing and fixing it while keeping the test code.

**Review by qtamu — APPROVED** [review]

**Comment by devangm** [comment]
Please, close this PR.

**Comment by rmathew1011** [comment]
Closing this PR

**Pull request closed without merging**